gpt4all-chat/CHANGELOG.md

```diff
@@ -4,6 +4,11 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.1.0/).
 
+## [Unreleased]
```

Comment:
This is a hyperlink. When you add one, it needs a referent at the bottom of the file:
```diff
+[Unreleased]: https://github.com/nomic-ai/gpt4all/compare/v3.9.0...HEAD
 [3.9.0]: https://github.com/nomic-ai/gpt4all/compare/v3.8.0...v3.9.0
```